modules/friendlywelcome.js

```diff
@@ -240,7 +240,7 @@ Twinkle.welcome.templates = {
 			},
 			'welcome-short': {
 				description: 'a shorter welcome message',
-				syntax: '{{subst:welcome-short|$USERNAME$}} $EXTRA$ ~~~~'
+				syntax: '$HEADER$ {{subst:w-short}} $EXTRA$ '
```

Comment:
This has a `heading` parameter, shouldn't we just use that?
```suggestion
				syntax: '{{subst:w-short|heading=yes}} $EXTRA$ '
```

Reply:
> This has a `heading` parameter, shouldn't we just use that?

Done